source/win-main/MainSidebar.vue

```diff
@@ -427,6 +428,25 @@ export default defineComponent({
         path: descriptor.path,
         id: descriptor.id // Convenience
       }))
+    },
+    /**
+     * Whether the cursor is within the corresponding document section
+     *
+     * @param   {number}  tocEntryLine          Line number of section heading
+     * @param   {number}  tocIdx                Index of heading in ToC
+     */
+    tocIsActive: function (tocEntryLine: number, tocIdx: number) {
+      const cursorLine = this.$store.state.activeDocumentInfo.cursor.line
+      // Determine index of next heading
+      const nextTocIdx = Math.min(tocIdx + 1, this.tableOfContents.length - 1)
+      var nextTocEntryLine = this.tableOfContents[nextTocIdx].line
+      // When this is the last heading, use Infinity as the next "heading"
+      if (nextTocEntryLine === tocEntryLine) {
+        nextTocEntryLine = Infinity
```

Comment:
I would simply preset that var with Infinity and then overwrite this iff the current ToC entry is not the last one. Plus, I think the linter is set to only accept let and const, but not var!

Reply:
Done in 41d94f8!